orc8r/cloud/go/obsidian/swagger/handlers/handlers.go

```diff
@@ -33,3 +66,115 @@
 		return c.String(http.StatusOK, combined)
 	}
 }
+
+// GetSpecHandler returns a routing handler which creates and
+// serves the raw YAML spec of a singular service.
+func GetSpecHandler(yamlCommon string) echo.HandlerFunc {
+	return func(c echo.Context) error {
+		requestedService := c.Param("service")
+
+		service, ok, err := getServiceName(requestedService)
+		if err != nil {
+			return obsidian.HttpError(err, http.StatusInternalServerError)
+		}
+		if !ok {
+			return obsidian.HttpError(errors.New("service not found"), http.StatusNotFound)
+		}
+
+		combined, err := swagger.GetCombinedSpecFromService(yamlCommon, service)
+		if err != nil {
+			return obsidian.HttpError(err, http.StatusInternalServerError)
+		}
+
+		return c.String(http.StatusOK, combined)
+	}
+}
+
+// SpecInfo defines the URL to be injected into
```

Comment:
The fact that this template variables struct contains specifically the URL info should be handled in the doc comment for the URL field